### PR TITLE
Register

### PR DESCRIPTION
**Pull request by ilzeLourens**

### CODE REVIEW

File: skietbaan_fe/src/components/Login.js [hunk 1]
```diff
@@ -23,17 +23,20 @@ class Login extends Component {
...
     if (target.name === "passwordValue" ) {
       stateUpdate.invalidPassword= false;
-    };
+    }
     if (target.name === "usernameValue" ) {
       stateUpdate.invalidUsername= false;
-    };
+      isValid = false;
+    }
+    if(this.state.usernameValue && this.state.passwordValue)
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/Login.js]: be consistent with if statements and their use of brackets `{ }`

File: skietbaan_fe/src/components/RegisterStyles.css [hunk 2]
```diff
@@ -50,22 +48,60 @@ input.invalid{
   color:red;
   font-size: 12px;
 }
+.header-container{
+  margin-left: -15px;
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/RegisterStyles.css]: inconsistent indentation

File: skietbaan_fe/src/components/RegisterStyles.css [hunk 3]
```diff
@@ -50,22 +48,60 @@ input.invalid{
...
+    padding-bottom: 5px;
+    padding-top: 5px;
+    text-align: center;
+
+}
+.front-white{
+  color:white
+}
+.button-container{
+  position: relative;
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/RegisterStyles.css]: inconsistent indentation and throughout the file

File: skietbaan_fe/src/components/Login.js (hunk 1 above)
Comment (ilzeLourens) [reply skietbaan_fe/src/components/Login.js]: fixed

File: skietbaan_fe/src/components/RegisterStyles.css (hunk 2 above)
Comment (ilzeLourens) [reply skietbaan_fe/src/components/RegisterStyles.css]: sorry fixed

File: skietbaan_fe/src/components/RegisterStyles.css (hunk 3 above)
Comment (ilzeLourens) [reply skietbaan_fe/src/components/RegisterStyles.css]: fixed

File: skietbaan_fe/src/components/Register.js [hunk 4]
```diff
@@ -167,8 +168,12 @@ class Register extends Component {
                 {invalidPasswordMessage}
               </FormGroup>
             </Col>
-            <Button onClick={this.Login} className={this.state.validForm ? "button-valid" : "button-invalid"} >Submit</Button>
+            <div className="button-container">
+            <Button onClick={this.Register} className={this.state.validForm ? "round-button" : "button-invalid round-button"} >Join</Button>
```
Comment (CilliersN) [inline skietbaan_fe/src/components/Register.js]: you can multiline this

Comment (CilliersN) [reply skietbaan_fe/src/components/Register.js]: <Button onClick={this.Register} 

className={this.state.validForm ? "round-button" : "button-invalid round-button"} >

Join

</Button>

Comment (CilliersN) [reply skietbaan_fe/src/components/Register.js]: just apply correct indentation

File: skietbaan_fe/src/components/RegisterStyles.css [hunk 5]
```diff
@@ -51,21 +49,65 @@ input.invalid{
...
-  background-color: #e0e1e2;
-  border: none;
+  color: #ebebeb!important;
+  background-color: #999!important;
+  border-color: #999!important;
+  cursor: default;
+}
+
+button.button-invalid:hover{
+  color: #ebebeb!important;
```
Comment (CilliersN) [inline skietbaan_fe/src/components/RegisterStyles.css]: oefh.... using !important everywhere makes messy and hard to maintain CSS.

File: skietbaan_fe/src/components/Register.js [hunk 6]
```diff
@@ -1,9 +1,9 @@
 import React, { Component } from 'react';
-import { Container, Col,FormGroup, Label, Input, Button, Form} from 'reactstrap';
+import { Container, Col, FormGroup, Label, Input, Button, Form } from 'reactstrap';
```
Comment (CilliersN) [inline skietbaan_fe/src/components/Register.js]: multiline

**Pull request merged**